connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java

```diff
@@ -208,6 +209,17 @@ public static String COMMIT_TASKS_KEY(String connectorName) {
             .field("creation-timestamp", Schema.INT64_SCHEMA)
             .build();
 
+    public static final String RESTART_PREFIX = "restart-connector-";
+
+    public static String RESTART_KEY(String connectorName) {
+        return RESTART_PREFIX + connectorName;
+    }
+
+    public static final Schema RESTART_REQUEST_V0 = SchemaBuilder.struct()
+            .field("include-tasks", Schema.BOOLEAN_SCHEMA)
+            .field("only-failed", Schema.BOOLEAN_SCHEMA)
```

Comment:
Should we use a constant for this? I see that it's not really a pattern in the existing code, but maybe we should still do that here.

Reply:
Good idea, I fixed it